test/find_xml_test.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'test_helper'
+
+describe "find_xml_file" do
+  it "should return nil if there are no xml files" do
+    expect(find_xml_file('.')).must_be_nil
```

Comment:
Could we assign the output of the method call to a variable and then expect that value of that variable to be nil?